manifests/0000_50_cluster-monitoring-operator_04-config.yaml

```diff
@@ -409,6 +409,58 @@ data:
     # consumers: (@openshift/openshift-team-cluster-manager)
     - '{__name__="console_url"}'
     #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_requests_total:sum gives the total number of login requests initiated from the web console.
+    #
+    - '{__name__="cluster:console_auth_login_requests_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_successes_total:sum gives the total number of successful logins initiated from the web console.
+    # Labels:
+    # * `role`, one of `kubeadmin`, `cluster-admin` or `developer`. The value is based whether or not the logged-in user can list all namespaces.
+    #
+    - '{__name__="cluster:console_auth_login_successes_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_login_failures_total:sum gives the total number of login failures initiated from the web console.
+    # This might include canceled OAuth logins depenending on the user OAuth provider/configuration.
+    # Labels:
+    # * `reason`, currently always `unknown`
+    #
+    - '{__name__="cluster:console_auth_login_failures_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_auth_logout_requests_total:sum gives the total number of logout requests send from the web console.
+    # Labels:
+    # * `reason`, currently always `unknown`
+    #
+    - '{__name__="cluster:console_auth_logout_requests_total:sum"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_usage_users:max contains the number of web console users splitten into the roles.
+    # Labels:
+    # * `role`: `kubeadmin`, `cluster-admin` or `developer`. The value is based whether or not the user can list all namespaces.
+    #
+    - '{__name__="cluster:console_usage_users:max"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_plugins_info:max reports information about the web console plugins and their state.
+    # Labels:
+    # * `name`: `redhat`, `demo` or `other`.
+    # * `state`: `enabled`, `disabled` or `notfound`
+    #
+    - '{__name__="cluster:console_plugins_info:max"}'
+    #
+    # owners: (@openshift/hybrid-application-console-maintainers)
+    # cluster:console_customization_perspectives_info:max reports information about customized web console perspectives.
```

Comment:
In the absolute worst case, yes.

It would surprise me to have more than a few clusters using this at all.

The most common case (on these few clusters) will be `admin` and `dev` will be `disabled`, `only-for-cluster-admins` or `only-for-developers`.

Reply:
So you're betting on a cardinality of 6?

Reply:
Correct, I think 4 to 6 is a reasonable assumption here for the clusters that use this feature.

The console returns just one state at each time, but the cluster admin can switch (change the config) this from `disabled` to `only-for-cluster-admins` or `only-for-developers` and back.